data/jsons/dependencies.json

```diff
@@ -0,0 +1,123 @@
+{
+    "dependencies": [
+	{
+	    "dependency": "check",
+	    "type": "pkg-config",
+	    "pkgname": "check"
```

Comment:
right, I was wondering if we *alread* have a user for that, talking to @glima we realized systemd would a first user... I'm writing that now. thanks.